server/src/main/resources/file.conf

```diff
@@ -1,4 +1,15 @@
 ## transaction log store, only used in seata-server
```

Comment:
move to line 13？

Reply:
As these parameters are for server thread pool instead of store, so we'd better put them out of the store section. 
这些参数是设置线程池的，最好还是放到server的段落，或者删除这一段，采用默认设置也是可以的。